test/security_test.rb

```diff
@@ -64,6 +79,15 @@ def enabled?; true; end
         end
       end
 
+      context "with services that are aliases of optional services" do
+        let(:service_names) { %w(ntp rsyslog auditd random kbd anacron postfix) }
```

Comment:
If you look at the whole file, ```%w``` is already used all along the test for defining the contexts. Probably it was considered acceptable for that particular use in the previous reviews.